### DISCUSSION TITLE
nouveau compte

### DISCUSSION THREAD

**Discussion by mnnomdutilisateur** [chamilo/chamilo-lms#5348] [General]
Bonjour Tout le monde,

J'ai été obligé de créé ce compte du fait qu'il m'est impossible d'ouvrir le compte que je possédais sur l'ancien forum (qui entre parenthèse était bien meilleur). Donc: Mon nom d'utilisateur sur l'ancien forum est "gaston", mais le nouveau forum n'en a rien a ciré. Même en changeant le mot de passe. Une solution Monsieur l'administrateur? 
P.S.: merci de m'indiquer le lien pour le forum en français

**Comment by yverhenne** [comment]
Bonsoir Gaston,

Comme indiqué sur l'ancien forum, il est maintenant hors service et reste disponible pour consultation. A la vue de ton message qui manque cruellement de contexte et de précision (qui entre parenthèse  est le cas aussi de tes différentes demandes d'aide sur l'ancien forum...), je ne comprend pas comment on peux t'aider?
Visiblement tu as réussi à créer un compte sur ce nouveau forum, qui n'en a rien a ciré, et donc tu peux poster (ce que tu as fait). C'est peut être que tu n'as pas le même nom d'utilisateur qui te tracasse, mais tu aurais pu mettre 'gaston' au lieu de 'mnnomdutilisateur' et aussi reprendre le même mot de passe. Ce qui aurait sans doute était la une bonne solution pour fluidifier ton utilisation. Tu peux toujours le faire également.
Si tu viens sur un forum ou un groupe c'est que tu es en recherche de solutions ou, face à des problématiques que tu ne sais pas résoudre seul, et afin de t'aider au mieux, il est nécessaire de bien préciser ta demande. 
Peux tu donc préciser ta demande afin que quelqu'un puisse y répondre au mieux Monsieur Gaston?
PS: ce forum est multilingues (quelques recherches aurait suffit à le comprendre)
Re PS : Je ne suis pas administrateur
